### PR TITLE
Added CCMenu

### PR DESCRIPTION
**Pull request by aurimasniekis**

### CODE REVIEW

File: Casks/ccmenu.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  url "https://github.com/erikdoe/ccmenu/releases/download/v#{version}/CCMenu.app.zip"
+  appcast 'https://github.com/erikdoe/ccmenu/releases.atom',
+          checkpoint: '91d810c7e4430bf642184dddf0ee54c47897ca5c23ab5d189467b071d9dad371'
+  name 'CCMenu'
+  homepage 'http://ccmenu.org/'
+  license :unknown # TODO: change license and remove this comment; ':unknown' is a machine-generated placeholder
+
+  app 'CCMenu.app'
+end
+
```
Comment (vitorgalvao) [inline Casks/ccmenu.rb]: There’s an extra line that should be removed. Other than that, looks great!

**Comment by vitorgalvao** [comment]
Extra note, you also have trailing whitespace in some lines (which is what is causing the build to fail).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in fc763e6558f0d3fe6bdd59f73b7f4d2775b61b66. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by gcds** [comment]
@vitorgalvao btw your style fixer has problem. It goes infinity loop and adds same line to same line on todo comment problem

**Comment by vitorgalvao** [comment]
Oh, right, had already seen that but forgot to open the issue. Thank you.

**Pull request closed without merging**